l10n/en-US/viewer.properties

```diff
@@ -280,3 +280,14 @@ editor_alt_text_save_button=Save
 editor_alt_text_decorative_tooltip=Marked as decorative
 # This is a placeholder for the alt text input area
 editor_alt_text_textarea.placeholder=For example, “A young man sits down at a table to eat a meal”
+
+# Editor resizers
+# LOCALIZATION NOTE (editor_resizer_label_***): This is used in an aria label to help to understand the role of the resizer.
```

Comment:
This doesn't work sadly. We need a comment for each string, right now this comment would only show up for the 1st in Pontoon